### ISSUE TITLE
Frontend: Investigate and fix accessibility issues

### ISSUE THREAD

**Issue by shadmau** [openPetition/opnVote#119]
## Overview

The following accessibility issues have been found and should be investigated/fixed

## Issues

- [ ] Election documents overview page: The actions for creating and loading the Security Key/Ballot cannot be reached via keyboard (Tab) --> deprecated or not incomprehensible (which browser was used?)
- [ ] The modal shown after Security Key creation cannot be closed via keyboard --> deprecated or not incomprehensible (which browser was used?)
- [ ] There is at least one missing or invalid `lang` attribute in the HTML structure, which could lead screen readers using the wrong language
- [ ] A button in the Security Key generation flow is missing an accessible name
